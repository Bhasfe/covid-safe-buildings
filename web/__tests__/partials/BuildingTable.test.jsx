import * as React from 'react';
import renderer from 'react-test-renderer';
import BuildingTable from '../../src/partials/buildings/BuildingsTable';

it('renders correctly', () => {
    const tree = renderer.create(<BuildingTable buildings={[]} />).toJSON();
    expect(tree).toMatchSnapshot();
});
