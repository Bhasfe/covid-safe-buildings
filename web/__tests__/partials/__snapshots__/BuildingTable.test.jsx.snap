// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="col-span-full xl:col-span-8 bg-white shadow-lg rounded-sm border border-slate-200"
>
  <header
    className="px-5 py-4 border-b border-slate-100"
  >
    <h2
      className="font-semibold text-slate-800"
    >
      Buildings
    </h2>
  </header>
  <div
    className="p-3"
  >
    <div
      className="overflow-x-auto"
    >
      <table
        className="table-auto w-full"
      >
        <thead
          className="text-xs uppercase text-slate-400 bg-slate-50 rounded-sm"
        >
          <tr>
            <th
              className="p-2"
            >
              <div
                className="font-semibold text-left"
              >
                Building
              </div>
            </th>
            <th
              className="p-2"
            >
              <div
                className="font-semibold text-center"
              >
                Safety Level
              </div>
            </th>
            <th
              className="p-2"
            >
              <div
                className="font-semibold text-center"
                title="Occupancy / Capacity"
              >
                Occup. / Cap.
              </div>
            </th>
            <th
              className="p-2"
            >
              <div
                className="font-semibold text-center"
              >
                Room Count
              </div>
            </th>
          </tr>
        </thead>
        <tbody
          className="text-sm font-medium divide-y divide-slate-100"
        />
      </table>
    </div>
  </div>
</div>
`;
